lib/src/widgets/utils/ds_group_card.widget.dart

```diff
@@ -1,14 +1,16 @@
 import 'package:blip_ds/blip_ds.dart';
-import 'package:blip_ds/src/utils/ds_message_content_type.util.dart';
 import 'package:blip_ds/src/widgets/chat/ds_message_bubble_detail.widget.dart';
+import 'package:blip_ds/src/widgets/utils/ds_card.widget.dart';
 import 'package:flutter/material.dart';
 
-/// A Design System widget used to display a grouped [DSMessageBubble] list
+/// A Design System widget used to display a grouped [DSCard] list
 class DSGroupCard extends StatefulWidget {
   final List<DSMessageItemModel> documents;
```

Comment:
In this case, shouldn't the quick reply be right-aligned?

<img width="333" alt="image" src="https://user-images.githubusercontent.com/753061/190689067-4d68d60a-60b9-4dd6-81c3-fdbdefd43f8b.png">

Reply:
no, it will always be on the left